open_spiel/algorithms/alpha_zero/README.md

```diff
@@ -1,6 +1,10 @@
-## Python AlphaZero
+# C++ AlphaZero
 
 This is a C++ implementation of the AlphaZero algorithm.
 
-For more information, please take a look at the
+For more information on the algorithm, please take a look at the
 [full documentation](https://github.com/deepmind/open_spiel/blob/master/open_spiel/docs/alpha_zero.md).
+
+[TensorflowCC library](https://github.com/mrdaliri/tensorflow_cc/tree/open_spiel) should be installed on your machine. Please see [this fork of tensorflow_cc](https://github.com/mrdaliri/tensorflow_cc/tree/open_spiel) for more building and installing instructions.
```

Comment:
You don't want to add it to install.sh? If it is still difficult, I understand that :)

Reply:
Well, it's not (at least yet) possible to install `tensorflow_cc` from a pre-packaged binary. Users have to pull it and then build it on their machines. Considering this process could be very time-consuming, I thought it would be better to keep it separate from OpenSpiel installer.

Reply:
Yes let's please keep C++ AlphaZero separate at this point since it requires a number of extra steps.